aixweather/core_data_format_2_output_file/to_epw_energyplus.py

```diff
@@ -629,4 +636,4 @@ def fill_full_last_day(df):
 
     logger.info("EPW file saved to %s.", file_path)
 
-    return df
+    return df, file_path
```

Comment:
Is it necessary to return the file_path?

I personally prefer a "as lean as possible" approach, but will most likely agree if you have arguments.

Reply:
I won't to have the file_path as a return value in the project-class either way. Having to build the filepath externally could lead to errors when you change the internal logic.

Reply:
Thank you for your answer. If I understand correctly, the file_path will not be returned in this PR?

Otherwise, please clarify.

Reply:
Yes, check the changes in project_class.py

Reply:
Currently the file path is returned in every composite function without being used (as far as I can interprete). Is there a reason to have the file path returned instead of silently assigning it to the output-file name?

Reply:
I wan't to use the filepath in other functions in my code, thus, I need the return.
The functions return it to the project class. which returns only the filepath:
```
    def core_2_json(self):
        """Convert core data to .json file"""
        self.output_data_df = to_json(
            self.core_data, self.meta_data, self.abs_result_folder_path
        Returns:
            str: Path to the exported file.
        """
        self.output_data_df, filepath = to_json(
            self.core_data, self.meta_data, self.abs_result_folder_path,
            filename=filename
        )
        return filepath
```

Reply:
Okay I understand! Good reason!